receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -148,7 +147,7 @@ func (ocr *Receiver) sendToNextConsumer(longLivedRPCCtx context.Context, traceda
 		receiverTransport)
 
 	// If the starting RPC has a parent span, then add it as a parent link.
-	observability.SetParentLink(longLivedRPCCtx, span)
+	obsreport.SetParentLink(longLivedRPCCtx, span)
```

Comment:
This change makes the code consistent with the recent change https://github.com/open-telemetry/opentelemetry-collector/blob/ea761567bdb475ef86e89416312db1df9db7e809/receiver/opencensusreceiver/ocmetrics/opencensus.go#L122